data/widgets/card/knowledgeDocument.ui

```diff
@@ -43,7 +43,7 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="expand">True</property>
-                <property name="valign">center</property>
+                <property name="valign">top</property>
```

Comment:
'top' does not exist, you probably meant something else. All of our apps show the following error:
```
Gtk-WARNING **: Failed to set property EknTableOfContents.valign to top: Could not parse enum: 'top'
```